.travis.yml

```diff
@@ -13,28 +14,41 @@ jobs:
     - stage: Tests
       name: "Test :skin: option"
       script:
-        - source env/conda/bin/activate sphinxcontrib-verilog-diagrams
         - cd tests && python3 -m unittest test.TestSkins
 
     - stage: Tests
       name: "Test :yosys_script: option"
       script:
-        - source env/conda/bin/activate sphinxcontrib-verilog-diagrams
         - cd tests && python3 -m unittest test.TestYosysScript
 
     - stage: Tests
       name: "Test verilog_diagram_yosys config variable"
       script:
-        - source env/conda/bin/activate sphinxcontrib-verilog-diagrams
         - cd tests && python3 -m unittest test.TestYosysType
 
+    - stage: Tests
```

Comment:
I'm not sure these should all be seperate stages?

Reply:
We have three stages for `sphinxcontrib-verilog-diagrams`:

1. `Tests` - checks whether the extensions work as intended
2. `Build` - checks the package building
3. `Deploy` deploys the package to the PyPI

Having separate jobs in the `Tests` stage lets them run in parallel, which is (in my opinion) generally good since it lowers the CI execution time, and gives you the opportunity to check which functionality is broken. If we increase the number of tests, we can also group them into several categories.

Reply:
Guess I'm still use to the old travis having a very slow set up time, hence it being better to have a few big builds rather than many small builds.

Reply:
@mithro Do you want me to combine the tests together, or we can merge this PR?